docs/xml-config-file.rst

```diff
@@ -156,3 +164,77 @@ too.
         <publisher bridge_name="file" publisher_name="file_publisher"/>
         <transformation file="libfile.so" function="addTimestamp"/>
     </connector>
+
+There are several possible types of connectors depending of the kind of its participants.
+Each connector type will refer to :ref:`example`.
+
+RTPS Bridge
+^^^^^^^^^^^
+
+In this kind of connector, both participant are RTPS compliant, like *shapes_projection* and *shapes_stereo* in our example file.
+
+.. image:: RTPS-bridge.png
+    :align: center
+
+RTPS to Other protocol
+^^^^^^^^^^^^^^^^^^^^^^
+
+This connector will communicate a RTPS environment with another protocol. Just like our *shapes_protocol* connector.
+
+Your *Bridge Library* must define at least a publisher to your desired protocol and it is responsible to
+communicate with it and follow the ISPublisher interface. By default, the transformation function is applied after
+:class:`on_received_data` method calls to the instance of ISBridge.
+If you want to change this behaviour you will need to override the complete data flow.
+
+*Bridge_configuration* node can contain configuration information that *Bridge Library* must understand.
+ISManager will parse the *property* nodes of each element and will call the respective :class:`create_`
+function of the library with a vector of pairs with the data contained.
+If no *bridge_configuration* is provided, then your createBridge will be called with :class:`nullptr` or an empty
+vector as parameter config.
+
+*Transformation* library could be reused by your bridge library, with the same or another transformation function inside the same transformation library (an example of reusing the transformation library can be found on `FIROS2 <https://github.com/eProsima/FIROS2/tree/master/examples/TIS_NGSIv2>`__.
+Of course, you can add built in transformation functions inside your *bridge library*.
+
+.. image:: IS-RTPS-to-Other.png
+    :align: center
+
+Other procotol to RTPS
```

Comment:
put xml example

Reply:
With "Each connector type will refer to :ref:example." I wanted to refer the bottom example, that, when clicked, opens the xml file.
Modified to "refer to the bottom :ref:example" for clearer reference.

Reply:
Added example xml.